### ISSUE TITLE
Wrong default Oracle JSON url in Ubuntu Focal build

### ISSUE THREAD

**Issue by MHova** [Cockatrice/Cockatrice#4565]
The default URL is `https://www.mtgjson.com/api/v5/AllPrintings.json.xz` (with the extra `.xz` at the end). Manually removing the `.xz` works.

**Comment by ZeldaZach** [comment]
It's possible your instance of Ubuntu doesn't support xzip by default. You can do .gz .bz2 .zip or just the .json and it should work.

**Comment by MHova** [comment]
Yes, I changed it to just plain .json and it worked. Should .xz not be the
default?

**Comment by ebbit1q** [comment]
please install `liblzma-dev` and recompile for .xz support, or simply use our precompiled binaries: https://github.com/Cockatrice/Cockatrice/releases

**Comment by MHova** [comment]
I installed today from the Focal binary at
https://github.com/Cockatrice/Cockatrice/releases/download/2021-01-26-Release-2.8.0/Cockatrice-Prismatic.Bridge-2.8.0-UbuntuFocal.deb

**Comment by ebbit1q** [comment]
then there's an entirely different issue where this binary does not include the required dependency for some reason then

**Comment by ebbit1q** [comment]
could you test if this issue is also present in the latest beta version?

**Comment by MHova** [comment]
Yeah sure can you point me to the beta release?

**Comment by ebbit1q** [comment]
https://github.com/Cockatrice/Cockatrice/releases/tag/2022-01-17-Development-2.8.1-beta.6

**Comment by ebbit1q** [comment]
do you have `liblzma5` installed?

**Comment by MHova** [comment]
Yes apparently so:
```
***@***:~$ apt list --installed liblzma5
Listing... Done
liblzma5/focal-updates,now 5.2.4-1ubuntu1 amd64 [installed,automatic]
liblzma5/focal-updates,now 5.2.4-1ubuntu1 i386 [installed,automatic]
```

**Comment by MHova** [comment]
The beta build does work. Let me try again with the latest release.

**Comment by MHova** [comment]
Hmm, the release build works with `.xz` as well. I recall punching the `.xz` url directly into the browser earlier today and it would not load. However, when I go there via the browser now, I am prompted to download the file.

I blame gremlins.

**Comment by ebbit1q** [comment]
good to hear it got sorted, maybe some left over configuration messed it up before

**Comment by MHova** [comment]
Or the .xz download was momentarily down for some reason? This was a
completely fresh install of Cockatrice (I'd never installed it before).

**Comment by ebbit1q** [comment]
there is a variety of network failures that could cause something like that, now that problem is gone we'll never know.
